customer-wallet/src/containers/Hoc/withCustomerTransactions.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react'
+import { object, func } from 'prop-types'
+// redux
+import { connect } from 'react-redux'
+import { getCustomerTransactions, updateCustomerTransactions } from 'store/modules/data/customerTransactions'
+// libs
+import { get, groupBy } from 'lodash'
+// components
+import CircularProgress from 'material-ui/CircularProgress'
+// assets
+import { address } from '../Customer/Transactions/assets/data'
+//
+const apiEnd = process.env.REACT_APP_BLOCKCYPHER_RESOURCE || 'test3'
+
+const withCustomerTransactions = (WrappedComponent) => {
+  class AsyncTransactionHistory extends Component {
+    static propTypes = {
+      customerTransactions: object,
+      getCustomerTransactions: func,
+      updateCustomerTransactions: func
+    }
+
+    socket = new WebSocket(`wss://socket.blockcypher.com/v1/btc/${apiEnd}`)
```

Comment:
@Venumteam Just want to know if you are listening for confirmations only for those transactions that have less than 6 confirmations. After that we shouldn't listen for confirmations.